specification/columns/availabilityzone.md

```diff
@@ -2,6 +2,16 @@
 
 An [*availability zone*](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
+---
+The AvailabilityZone column adheres to the following requirements:
+
+* AvailabilityZone is RECOMMENDED to be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports deploying resources or services within an *availability zone*.
+* If present, the column MUST conform to the following additional requirements:
```

Comment:
```suggestion
* If present, the column adheres to the following additional requirements:
```

Reply:
suggested change (MUST conform -> adheres) applied across the board